assets/css/menu/menu.css

```diff
@@ -0,0 +1,48 @@
+.sidenav {
```

Comment:
Try to follow pattern. There is no menu js file in this project. Create menu folder in css/components/ui.